.tekton/Dockerfile.hermetic

```diff
@@ -0,0 +1,43 @@
+# Multi-stage build for hermetic frontend build
+# Stage 1: Build stage
+FROM registry.access.redhat.com/ubi9/nodejs-22:9.5-1746535891 AS builder
+
+# Copy package files first for better caching
+COPY package*.json ./
+
+# Install dependencies using npm ci for hermetic builds
+# Set Node.js memory limit and npm options for hermetic environment
+ENV NODE_OPTIONS="--max-old-space-size=4096"
+# Skip Cypress binary download in hermetic environment
+ENV CYPRESS_INSTALL_BINARY=0
+# Use prefetched dependencies from Cachi2 in hermetic environment
+RUN npm ci --offline --no-audit --no-fund
+
+# Copy source code
+COPY . .
+
+# Run the build process
+RUN npm run build
+
+# Run tests and linting
+RUN npm run test
+RUN npm run lint
+
+# Stage 2: Minimal runtime image - contains only built files
+FROM scratch
+
+# Required Red Hat labels for container compliance
+LABEL com.redhat.component="landing-page-frontend-hermetic" \
+      description="Red Hat Console Landing Page Frontend - Hermetic Build" \
+      distribution-scope="public" \
+      io.k8s.description="Red Hat Console Landing Page Frontend built with hermetic process" \
+      name="landing-page-frontend-hermetic" \
+      release="1" \
+      url="https://github.com/RedHatInsights/landing-page-frontend" \
+      vendor="Red Hat, Inc." \
+      version="1.0" \
+      maintainer="Red Hat Console Team <console@redhat.com>" \
+      summary="Red Hat Console Landing Page Frontend"
+
+# Copy built files from builder stage
+COPY --from=builder /opt/app-root/src/dist /dist 
```

Comment:
Do we also need the lock file to be present?

Reply:
I'm not sure, but we could add it and the package.json as well. It shouldn't hurt